backend/drm/drm.c

```diff
@@ -1309,6 +1309,9 @@ void scan_drm_connectors(struct wlr_drm_backend *drm) {
 
 			if (curr_enc) {
 				wlr_conn->old_crtc = drmModeGetCrtc(drm->fd, curr_enc->crtc_id);
+			} else {
+				wlr_log(WLR_DEBUG, "Connector %d has no encoder",
```

Comment:
This doesn't really mean a lot. There's no encoder right now, but we could find an encoder later. I'd rather remove this log.

Reply:
Ping